.NET/Microsoft.Recognizers.Text.DateTime/Parsers/CJK/BaseCJKDateTimePeriodParser.cs

```diff
@@ -232,42 +232,24 @@ private DateTimeResolutionResult MergeTwoTimePoints(string text, DateObject refe
             DateObject futureBegin = (DateObject)((DateTimeResolutionResult)pr1.Value).FutureValue,
                 futureEnd = (DateObject)((DateTimeResolutionResult)pr2.Value).FutureValue;
 
-            DateObject pastBegin = (DateObject)((DateTimeResolutionResult)pr1.Value).PastValue,
-                pastEnd = (DateObject)((DateTimeResolutionResult)pr2.Value).PastValue;
+            DateObject pastBegin = (DateObject)((DateTimeResolutionResult)pr1.Value).PastValue;
 
             if (futureBegin > futureEnd)
             {
                 futureBegin = pastBegin;
             }
 
-            if (pastEnd < pastBegin)
-            {
-                pastEnd = futureEnd;
-            }
-
             if (bothHaveDates)
             {
                 rightTime = DateObject.MinValue.SafeCreateFromValue(futureEnd.Year, futureEnd.Month, futureEnd.Day);
                 leftTime = DateObject.MinValue.SafeCreateFromValue(futureBegin.Year, futureBegin.Month, futureBegin.Day);
             }
             else if (beginHasDate)
             {
-                // TODO: Handle "明天下午两点到五点"
```

Comment:
Do we have test cases tha show both of the examples in the deleted comments are handled correctly?

Reply:
yes I added a test case to show that the example is supported